cpp/memilio/mobility/graph_simulation.h

```diff
@@ -171,7 +171,7 @@ class GraphSimulationStochastic
                     size_t event = mio::DiscreteDistribution<size_t>::get_instance()(m_rng, m_rates);
                     //edge that performs transition event
                     auto& event_edge = Base::m_graph.edges()[event / parameters_per_edge];
-                    //index for compartment and age group moving
+                    //index for compartment and age group migrating
```

Comment:
```suggestion
                    //index for compartment and age group moving
```
We tried to remove migrate/migration and replace it with mobility/movement.